### ISSUE TITLE
SegmentationFault at 139935320384573

### ISSUE THREAD

**Issue by jub0t** [oven-sh/bun#337]
I was just trying out Bun when this happened.

```sh
jub0t@jub0t:~/Other/TestCoding$ bun index.ts

SegmentationFault at 139935320384573


–––– bun meta ––––
Bun v0.1.1 Linux x64 #43-Ubuntu SMP Wed Jun 15 12:54:21 UTC 2022
AutoCommand: 
Elapsed: 344ms | User: 321ms | Sys: 142ms
RSS: 71.31MB | Peak: 55.03MB | Commit: 71.31MB | Faults: 0
–––– bun meta ––––

Ask for #help in https://bun.sh/discord or go to https://bun.sh/issues
```
```ts
import fs from 'fs'

fetch('https://api.github.com/users/octocat')
  .then((res) => res.json())
  .then((data) => {
    console.log(data)
  })
```

**Comment by Jarred-Sumner** [comment]
Thank you. I can reproduce this.

**Comment by meister03** [comment]
having the same issue, did someone found a work around to bypass this issue

**Comment by FinnRG** [comment]
Resolved with version ``0.1.4``.
